pkg/resource/db_instance/hooks.go

```diff
@@ -14,10 +14,27 @@
 package db_instance
 
 import (
+	"context"
 	"errors"
 	"fmt"
 
+	svcapitypes "github.com/aws-controllers-k8s/rds-controller/apis/v1alpha1"
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
+	ackcondition "github.com/aws-controllers-k8s/runtime/pkg/condition"
 	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
+	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
+	svcsdk "github.com/aws/aws-sdk-go/service/rds"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Hack to avoid import errors during build...
+var (
+	_ = &metav1.Time{}
+	_ = &svcsdk.RDS{}
+	_ = &svcapitypes.DBInstance{}
+	_ = ackv1alpha1.AWSAccountID("")
+	_ = &ackcondition.NotManagedMessage
```

Comment:
I don't think this is necessary for hook files. We can optionally import if we need these packages.

Reply:
synced offline, addressed in next revision